cita-auth/src/handler.rs

```diff
@@ -1090,3 +1094,24 @@ fn snapshot_response(sender: &Sender<(String, Vec<u8>)>, ack: Resp, flag: bool)
         ))
         .unwrap();
 }
+
+// min quota is 21000 (or 53000 if tx.to == 0) + (4 per byte)
+// tx quota: 21000
+// create contract: 53000
+// only verify if tx.version > 2
+pub fn verify_min_quota(tx: &Transaction) -> bool {
+    match tx.get_version() {
+        0...2 => true,
+        _ => {
+            let quota = tx.get_quota();
+            let to = tx.get_to_v1();
+            if to.is_empty() || Address::from(to) == Address::zero() {
+                let min_quota = 53000 + 4 * tx.data.len();
```

Comment:
ref: https://github.com/cryptape/cita/blob/develop/cita-executor/evm/src/schedule.rs#L100
The quota for create a contract, is
```
tx_create_gas: 53_000
```
but not `32000`.
`32000` is the quota for `CREATE` instruction.

BTW, don't use the number like `21000` and `32000` directly, it is better to define a Macro or use `let schedule = Schedule::new_v1();` for it.

Reply:
Struct Schedule is in the evm crate, maybe it's no worth to use it here.
I don't know how to define a Macro in this case.